website/registries/views.py

```diff
@@ -19,13 +19,14 @@ def _view_registries_landing_page(campaign=None, **kwargs):
     else:
         has_projects = False
 
+    campaign_url_param = 'osf-registered-reports' if campaign == 'registered_report' else 'prereg'
```

Comment:
For more future-proofing,  I wouldn't assume that our only campaigns are either 'registered_report' or 'prereg', especially having prereg be the catch-all for all types of campaigns.  We could add other campaigns in the future.

I'd recommend an if/elif.  if campaign == 'registered_report': ...elif campain == 'prereg_challenge' ... If it's not registered reports, and if it's not prereg, then we're not adding a campaign query param.